WordPress/src/main/java/org/wordpress/android/ui/reader/discover/ReaderDiscoverViewModel.kt

```diff
@@ -145,6 +144,7 @@ class ReaderDiscoverViewModel @Inject constructor(
                         }
                     }
                 }
+                trackContentPresentedOnce()
```

Comment:
❓ This will be tracked even when we load data from cache. I believe it won't help us answer `We want to check if the API being slow is affecting Reader usage in the Discover section. The how is by using two events in a funnel.`. Thoughts @ParaskP7 @leandroalonso ?

Reply:
I am not sure about this, maybe in iOS, because they have no cache data.

But, in our case, what I want to achieve is based on the content shown, not the data arriving. So, if the content is shown, either through the API or cache, I am okay with that, as the result will be the same for the user, their content being shown. On the other hand, if the content is not shown, because of the API being slow, or maybe because of cache problems (doubt this will happen), then I want to track that. Wdyt?

Reply:
@ParaskP7 I'll only trigger that if a request has been made. Otherwise, as @malinajirka mentioned we won't be able to answer the main question.

So:

1. Request triggered -> Content rendered: fire the event
2. Content rendered: don't fire

Again, just like mentioned below, if you both think the event name is misleading, we can change it. :)

Reply:
What you are proposing @leandroalonso is quite difficult to do in our implementation and I'm not sure it's worth it.

- ViewModel has no idea where the data come from (cache or server)
- DataProvider and API layer have no idea what is on the UI

Moreover, this will come down to -> track this event only after the first installation if the user didn't leave before the content was shown. Is this really what we want to track?

Reply:
I agree with Jirka @leandroalonso , in the Android case, the way it is implemented, this layer is agnostic of where the data came from, and for a good reason. To me, since what we want to understand, is when the user leaves the screen, before the content gets presented to them, we shouldn't care how this is done on the data layer, we should only care about the end result, content being presented or not. Thus, I am leaving the API out of it, being an implementation detail. Wdyt?